boards/uvify/core/default.cmake

```diff
@@ -0,0 +1,115 @@
+
+px4_add_board(
+	PLATFORM nuttx
+	VENDOR uvify
+	MODEL core
+	LABEL default
+	TOOLCHAIN arm-none-eabi
+	ARCHITECTURE cortex-m4
+	ROMFSROOT px4fmu_common
+	TESTING
+	UAVCAN_INTERFACES 1
+
+	SERIAL_PORTS
+		GPS1:/dev/ttyS3
+		TEL1:/dev/ttyS1
+		TEL2:/dev/ttyS2
+
+	DRIVERS
+		barometer # all available barometer drivers
+		batt_smbus
+		camera_capture
+		camera_trigger
+		differential_pressure # all available differential pressure drivers
+		distance_sensor # all available distance sensor drivers
+		gps
+		heater
+		imu # all available imu drivers
+		irlock
+		lights/blinkm
+		lights/oreoled
+		lights/rgbled
+		lights/rgbled_ncp5623c
+		magnetometer # all available magnetometer drivers
+		mkblctrl
+		optical_flow # all available optical flow drivers
+		pca9685
+		pwm_input
+		pwm_out_sim
+		px4fmu
+		rc_input
+		stm32
+		stm32/adc
+		stm32/tone_alarm
+		tap_esc
+		telemetry # all available telemetry drivers
+		test_ppm
+		tone_alarm
+		uavcan
+
+	MODULES
+		attitude_estimator_q
+		camera_feedback
+		commander
+		dataman
+		ekf2
+		events
+		fw_att_control
```

Comment:
Do you only ever use this board on multicopters (draco, etc)? If so you could comment out the FW, GND, and VTOL controllers.

Reply:
None MC controllers commented in default.